chapters/synchronous.tex

```diff
@@ -1400,13 +1400,13 @@ \subsection{Inferencing of solverMethod}\label{inferencing-of-solvermethod}
 
 \section{Initialization of Clocked Partitions}\label{initialization-of-clocked-partitions}
 
-The standard scheme for initialization of Modelica models does not apply for clocked base-partitions.
+The standard scheme for initialization of Modelica models does not apply for discrete-time sub-partitions.
 Instead, initialization is performed in the following way:
 \begin{itemize}
 \item
-  Variables in clocked partitions cannot be used in initial equation or initial algorithm sections.
+  Variables in discrete-time sub-partitions cannot be used in initial equation or initial algorithm sections.
 \item
-  Attribute \lstinline!fixed! cannot be applied to variables in clocked partitions.
+  Attribute \lstinline!fixed! cannot be applied to variables in discrete-time sub-partitions.
```

Comment:
Should we really allow `fixed` to be applied to clocked states in clocked discretized continuous-time partitions?  Wouldn't that conflict with `fixed` already being defined for these variables?

Reply:
It shouldn't be a conflict! Instead they should keep their "earlier" fixed-ness. (Technically we cannot know whether the fixed-attribute was set before or after it was discretized.) Exactly how that works isn't clear in general.